elide-contrib/elide-test-helpers/src/test/java/com/yahoo/elide/contrib/testhelpers/graphql/GraphQLDSLTest.java

```diff
@@ -0,0 +1,266 @@
+                selections(
+                        field(
+                                QUERY,
+                                "book",
+                                arguments(
+                                        argument("sort", "-id", QUOTE_VALUE),
+                                        argument("id", "5", QUOTE_VALUE)
+                                ),
+                                selections(
+                                        field("id"),
+                                        field("title")
+                                )
+                        )
+                )
+        ).toQuery();
+
+        Assert.assertEquals(actual, expected);
+    }
+
+    @Test
+    public void verifyRequestWithVariable(){
+        String expected = "query myQuery($bookId: [String]) {book(ids: $bookId) {edges {node {id title authors {edges" +
+                " {node {name}}}}}}}";
+
+        String actual = document(
+                query(
+                        "myQuery",
+                        variableDefinitions(
+                                variableDefinition("bookId", "[String]")
+                        ),
+                        selections(
+                                field(
+                                        QUERY,
+                                        "book",
+                                        arguments(
+                                                argument("ids", "$bookId")
+                                        ),
+                                        selections(
+                                                field("id"),
+                                                field("title"),
+                                                field(
+                                                        QUERY,
+                                                        "authors",
+                                                        selection(
+                                                                field("name")
+                                                        )
+                                                )
+                                        )
+                                )
+                        )
+                )
+        ).toQuery();
+
+        Assert.assertEquals(actual, expected);
+    }
+
+    /**************************************** Response DSL ****************************************/
+
+    @Test
+    public void verifyBasicResponse() {
+        String expected = "{\"data\":" +
+                "{\"book\":" +
+                "{\"edges\":" +
+                "[{\"node\":" +
+                "{\"id\":\"1\"," +
+                "\"title\":\"My first book\"," +
+                "\"authors\":{\"edges\":[{\"node\":{" +
+                "\"name\":\"Ricky Carmichael\"}}]}}}]}}}";
+
+        String actual = document(
+                selection(
+                        field(
+                                RESPONSE,
```

Comment:
I don't feel like we need RESPONSE defined here and also a separate method called toResponse.  Just use the latter and remove this extra field parameter.